_hackbars/zoltan_karpati.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: Open Source
+author: Tasnim Zotder
```

Comment:
@itkarpathy Can you please update the `author` field? 👀

Reply:
Yes, sure. I changed for that. Thanks!